### PR TITLE
Add compat data for document.fonts

### PR DESCRIPTION
**Pull request by riverspirit**
Add compat data for [document.fonts](https://developer.mozilla.org/en-US/docs/Web/API/Document/fonts).

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -3485,6 +3485,58 @@
           }
         }
       },
+      "fonts": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fonts",
+          "support": {
+            "chrome": {
+              "version_added": "35"
```
Comment (jpmedley) [inline api/Document.json]: Chrome 60 https://storage.googleapis.com/chromium-find-releases-static/32d.html#32dbbeab5cdae35fb757924d339b2d83bc4ea487

File: api/Document.json [hunk 2]
```diff
@@ -3485,6 +3485,58 @@
...
+            },
+            "chrome_android": {
+              "version_added": "69"
```
Comment (jpmedley) [inline api/Document.json]: 60

File: api/Document.json [hunk 3]
```diff
@@ -3485,6 +3485,58 @@
...
+              "notes": "First implemented in Firefox 35 behind the flag <code>layout.css.font-loading-api.enabled</code>. Enabled by default in Firefox 41."
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
```
Comment (jpmedley) [inline api/Document.json]: 47

File: api/Document.json [hunk 4]
```diff
@@ -3485,6 +3485,58 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Document.json]: 47

File: api/Document.json [hunk 5]
```diff
@@ -3485,6 +3485,58 @@
...
+            },
+            "safari": {
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10.2"
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Document.json]: 60

This should also be after Samsung.

**Comment by riverspirit** [comment]
Thanks for the corrections @jpmedley. I had taken the values from [caniuse](https://caniuse.com/#feat=font-loading). I've corrected the commit now.

File: api/Document.json [hunk 6]
```diff
@@ -3485,6 +3485,58 @@
...
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "First implemented in Firefox 35 behind the flag <code>layout.css.font-loading-api.enabled</code>. Enabled by default in Firefox 41."
```
Comment (connorshea) [inline api/Document.json]: This should be represented using the `flags` attribute instead: https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags

Like this:

```json
"firefox": [
  {
    "version_added": "41"
  },
  {
    "version_added": "35",
    "version_removed": "41",
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.font-loading-api.enabled",
        "value_to_set": "true"
      }
    ]
  }
]
```

Comment (riverspirit) [reply api/Document.json]: Thanks for the pointer @connorshea. I've updated the PR.

**Pull request merged**